arches_lingo/src/arches_lingo/components/generic/LabelViewer.vue

```diff
@@ -0,0 +1,160 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import DataTable from "primevue/datatable";
+import Column from "primevue/column";
+import Button from "primevue/button";
+import ConfirmDialog from "primevue/confirmdialog";
+import { useConfirm } from "primevue/useconfirm";
+
+import type { AppellativeStatus } from "@/arches_lingo/types";
```

Comment:
nit: types are bottom-most imports

Reply:
Would be great if we had a style guide for documenting the preferred order of imports.